src/soep_preparation/create_metadata/task.py

```diff
@@ -98,31 +105,31 @@ def task_create_metadata(
         }
 
 
-@task(after="task_create_metadata")
+MAPPING_NAME_TO_METADATA = _create_name_to_metadata_mapping(MAPPING_NAME_TO_DATA.keys())
+
+
 def task_create_metadata_mapping(
-    single_metadata_mapping: Annotated[dict, DATA_CATALOGS["metadata"]],
+    mapping_name_to_metadata: Annotated[
```

Comment:
```suggestion
    map_name_to_metadata: Annotated[
```
Same for `map_variable_to_data_file`